cargo-dist/src/tasks.rs

```diff
@@ -1954,6 +1960,9 @@ impl<'pkg_graph> DistGraphBuilder<'pkg_graph> {
             return;
         };
         let bin_aliases = release.bin_aliases.for_targets(&target_triples);
+
+        let runtime_conditions = pick_runtime_conditions(&self.manifest.systems);
```

Comment:
Why can't this be computed centrally in platform_support?

Reply:
Specifically this info would ideally be embedded in `artifacts`

Reply:
Ah I see right there's no more code between here and the template... Ok so I propose that the "right" information be computed centrally in platform_support and then we have another "compatibility method" like .fragments() that does the flattening to runtime_conditions.